bot/app/database/db_objects.py

```diff
@@ -230,4 +217,62 @@ def all_users(self):
         conn.close()
         curs.close()
 
-        return users
\ No newline at end of file
+        return users
+
+def get_next_event():
+
+    """Retrieves the next upcoming event from the db, returns an Event object
+    """
+
+    conn = sqlite3.connect(DB_NAME)
+    curs = conn.cursor()
+    
+    curs.execute("""select max(eventID) from Events""")
+    event_id = curs.fetchone()
+    
+    curs.execute("""select * from Events where eventID=?""", event_id)
+    next_event = curs.fetchone()
+    
+    curs.close()
+    conn.close()  
+    
+    return next_event
```

Comment:
You should probably parse this into an `Event` class, then you can access it by it's attributes, rather than by a list index.

Reply:
fixed?